drivers/usb/device/usb_dc_mcux.c

```diff
@@ -14,6 +14,7 @@
 #include <soc.h>
 #include <zephyr/device.h>
 #include <zephyr/kernel.h>
+#include <zephyr/drivers/pinctrl.h>
```

Comment:
Add `select PINCTRL` to the Kconfig that enables the USB driver, as this is how we insure pin control is enabled in other drivers: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/spi/Kconfig.mcux_lpspi#L11